src/community/script/groovy/pom.xml

```diff
@@ -27,10 +33,10 @@
    <version>1.7-SNAPSHOT</version>
   </dependency>
   <dependency>
-    <!-- this dependency taken from geoscript-groovy pom but upgraded to 1.0.8 -->
+    <!-- this dependency taken from geoscript-groovy pom but upgraded to 1.0.9 -->
     <groupId>no.ecc.vectortile</groupId>
     <artifactId>java-vector-tile</artifactId>
-    <version>1.0.8</version>
+    <version>1.0.9</version>
```

Comment:
Based on https://github.com/geoserver/geoserver/blob/master/src/extension/vectortiles/pom.xml#L17 , apparently not?